apps/gitness/src/pages-v2/repo/repo-list.tsx

```diff
@@ -50,6 +52,7 @@ export default function ReposListPage() {
       retry: 5
     }
   )
+  const PAGE_SIZE = parseInt(headers?.get(PageResponseHeader.xPerPage) || '10')
```

Comment:
why default to 10? that's a very small number. also, it wasn't added earlier, so that means BE was choosing the default?